Makefile

```diff
@@ -210,6 +210,17 @@ else
 endif
 	@git push --force origin refs/tags/test:refs/tags/test
 
+tag-prod:
```

Comment:
Automatically tag version when we tag prod?

Reply:
yes